code/game/objects/effects/landmarks/landmarks.dm

```diff
@@ -425,8 +425,6 @@
 
 /obj/effect/landmark/sensor_tower/Initialize()
 	. = ..()
-	var/area/area_to_control = get_area(src)
-	area_to_control.set_to_contested()
```

Comment:
dead proc now

Reply:
I didn't want to remove all the sensor tower stuff in case it gets revived or something, but yeah they can just add it back in. I'll remove.